config/suppressions.xml

```diff
@@ -6,7 +6,7 @@
 
 <suppressions>
   <!-- can't split long messages between lines -->
-  <suppress checks="RegexpSingleline" files="google_checks\.xml" lines="56,126"/>
+  <suppress id="lineLengthXml" files="google_checks\.xml" lines="56,126"/>
   <!-- don't validate generated files -->
   <suppress checks="RegexpSingleline" files="releasenotes\.xml"/>
```

Comment:
Can we use id there ?

It would be awesome to not have in code lines that match
`<suppress checks="Regexp`
at all

Can we configure just another Regexp Check to control this ?

Reply:
> Can we use id there ?

We can. It might lead to just one line suppression expanding into multiple lines.

> Can we configure just another Regexp Check to control this ?

Definitely. You might want to create a new issue for this.

Reply:
https://github.com/checkstyle/checkstyle/issues/8886